### PR TITLE
Azure Deployment Environments blog post

### PR DESCRIPTION
**Pull request by mikhailshilkov**
### Proposed changes

Azure Deployment Environments blog post

### CODE REVIEW

File: content/blog/azure-deployment-environments/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+authors:
+    - mikhail-shilkov
+tags:
+    - azure
+    - announcement
+    - platform-engineering
+---
+
+We are excited to announce the support for authoring [Azure Deployment Environments (ADE)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-environments) environment definitions in Pulumi Infrastructure as Code (IaC) empowering developers to self-serve app infrastructure required to deploy and test cloud-based applications. With Pulumi support, you can now manage your Azure resources in these environments using the same familiar programming model and the full power of our IaC platform.
+
```
Comment (desteves) [inline content/blog/azure-deployment-environments/index.md:17]: Consider adding the 
```html 
<!--more-->
``` 
here instead per the [blogging suggestion checklist item](https://github.com/pulumi/docs/blob/master/BLOGGING.md#publishing-check-list)

File: content/blog/azure-deployment-environments/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+meta_image: meta.png
+authors:
+    - mikhail-shilkov
+tags:
+    - azure
+    - announcement
+    - platform-engineering
+---
+
+We are excited to announce the support for authoring [Azure Deployment Environments (ADE)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-environments) environment definitions in Pulumi Infrastructure as Code (IaC) empowering developers to self-serve app infrastructure required to deploy and test cloud-based applications. With Pulumi support, you can now manage your Azure resources in these environments using the same familiar programming model and the full power of our IaC platform.
```
Comment (desteves) [inline content/blog/azure-deployment-environments/index.md:16]: Link is returning 404

File: content/blog/azure-deployment-environments/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+If you have a suggestion for improving the standard Pulumi container image, please [open an issue](https://github.com/pulumi/azure-deployment-environments/issues/new) on our GitHub repository. We are always looking for ways to improve the experience for our Azure users.
+
+## Conclusion
+
+Together with the Azure Deployment Environments team, Pulumi is excited to enable our shared customers to take advantage of the ADE extensibility model, Pulumi IaC platform, and empower developers with self-service of Azure App infrastructure, within enterprise guardrails. We hope that this new capability will help you streamline your development processes and make it easier for your teams to provision new environments in Azure quickly.
+
+If you want to learn more:
+- Attend our workshop [Platform Engineering with Microsoft Azure and Pulumi](https://bit.ly/4dwyrIzp) that takes place on June 20
```
Comment (desteves) [inline content/blog/azure-deployment-environments/index.md:89]: Bit.ly is not working

File: content/blog/azure-deployment-environments/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+Finally, **navigate to the [Microsoft developer portal](https://devportal.microsoft.com)** to create a new environment. You will see the list of available templates, including the ones you just published.
+
+## Bring a Custom Container Image
+
+The Pulumi container image [`pulumi/azure-deployment-environments`](https://hub.docker.com/repository/docker/pulumi/azure-deployment-environments) is a great starting point for your environment definitions. However, if you want to customize the behavior to your needs, you can also bring your own container image to run Pulumi programs in ADE. To do so, refer to the [Configure a container image to execute deployments with Pulumi](https://aka.ms/ade/pulumi-docs) article on Microsoft Learn.
```
Comment (desteves) [inline content/blog/azure-deployment-environments/index.md:78]: Link is not working.

Comment (mikhailshilkov) [reply content/blog/azure-deployment-environments/index.md:78]: Yes, they will publish later today, same as us

**Review by desteves — APPROVED** [review]

**Pull request merged**